bindings/zig/src/opendal.zig

```diff
@@ -1,13 +1,29 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
 pub const opendal = @cImport(@cInclude("opendal.h"));
 
 const std = @import("std");
 const testing = std.testing;
 
 test "Opendal BDD test" {
-    const c_str = [*:0]const u8; // const char*
-    // const zig_str = []const u8;
+    const c_str = [*:0]const u8; // define a type for 'const char*' in C
 
-    const OpendalBddTest = struct {
+    const OpendalBDDTest = struct {
```

Comment:
The next step could be factored out as a first-class Zig struct to work with. This test kit demonstrates how we will use it.

Reply:
Mark as resolved to pass the state check.